webpack.config.common.js

```diff
@@ -2,7 +2,6 @@ const path = require('path');
 const webpack = require('webpack');
 const CopyPlugin = require('copy-webpack-plugin');
 const LicenseWebpackPlugin = require('license-webpack-plugin').LicenseWebpackPlugin;
-const WriteBundleFilePlugin = require('./config/WriteBundleFilePlugin');
 const BundleAnalyzerPlugin = require('webpack-bundle-analyzer').BundleAnalyzerPlugin;
```

Comment:
The related `config/WriteBundleFilePlugin.js` file can probably be deleted now since we are not using it (the plugin) any more.

Reply:
Good point, I'll remove it.